src/components/TopBar.tsx

```diff
@@ -69,8 +69,8 @@ export default function TopBar(props: Props): JSX.Element {
     <Container>
       <Content>
         {backButton && (
-          <BackIconLink onClick={() => Router.back()}>
-            <BackIcon />
+          <BackIconLink onClick={() => Router.back()} data-test="top-bar-back-icon">
```

Comment:
What does this do?

Reply:
It's used to select the element for testing. The purpose is to allow the element to be selected independent of style and nesting changes. (https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements)